.pre-commit-hooks.yaml

```diff
@@ -0,0 +1,7 @@
+- id: verilator
+  name: verilator-lint
+  description: Runs verilator Docker image to lint (System) Verilog designs
+  args: [--lint-only, -Wall]
```

Comment:
`-Wall` doesn't change the gating and warnings won't be shown at all when there are no errors. I'm adding `-Wall` because the help message gives that as an example. Is there a way of treating all warnings as errors?

Reply:
Verilator by default treats all warning as errors. On one hand, I think Wall is reasonable to include, users can disable individual warnings if they prefer, but on the other it might be a surprise this is part of what docker does versus default Verilator behavior - so maybe better just to have lint-only?

Reply:
Point taken, let's

```suggestion
  args: [--lint-only]
```